build.rs

```diff
@@ -1,9 +1,15 @@
+use std::env::var;
+
 fn main() {
     #[cfg(feature = "user")] {
         if std::env::var("TARGET").map(
```

Comment:
```suggestion
        if var("TARGET").map(
```
This can be removed now